content/en/integrations/guide/deprecated-oracle-integration.md

```diff
@@ -0,0 +1,534 @@
+---
+title: Configuring Oracle integration on the Agent versions lower than 7.50.1
+kind: guide
+aliases:
+  - integrations/guide/deprecated-oracle-integration
+---
+
+# Oracle Integration
+
+![Oracle Dashboard][1]
```

Comment:
```suggestion
# Oracle Integration

![Oracle Dashboard][1]
```
Let's remove the dashboard image. The guide probably just needs the old setup steps